prov/rxm/src/rxm_conn.c

```diff
@@ -433,6 +433,7 @@ void rxm_cmap_process_shutdown(struct rxm_cmap *cmap,
 			"Invalid handle on shutdown event\n");
 	} else if (handle->state != RXM_CMAP_SHUTDOWN) {
 		FI_DBG(cmap->av->prov, FI_LOG_EP_CTRL, "Got remote shutdown\n");
+		rxm_cmap_del_handle(handle);
```

Comment:
We also have the option to continue to defer the deletion of the handle if the EP is bound to an SRX, since RX buffers would not be lost. I don't know that it is necessary, justing documenting it would be possible to do so.